### DISCUSSION TITLE
QE including virials data

### DISCUSSION THREAD

**Discussion by saeedshahi** [deepmodeling/dpdata#213] [Q&A]
Hi all,

I have a dataset that is calculated by QE (CPMD) and includes water.evp, water.for, water.str (stress tensor), water.pos, water.cel files. When I want to convert them to npy they show me Including Virials: No. I think stress tensor is that virials. Is that right? what is wrong? which file in QE includes the virials? 

Best,
Saeed

**Comment by wanghan-iapcm** [comment]
Did you you `dpdata` for the conversion?

**Reply by saeedshahi** [reply]
Yes

**Comment by amcadmus** [comment]
@robinzyb Could you please take a look? Thanks!
